Cargo.toml

```diff
@@ -68,6 +68,7 @@ axum-extra = "0.9.3"
 axum-server = "0.7.1"
 arc-swap = "1.7.1"
 backoff = "0.4.0"
+bb8-redis = "0.17.0"
```

Comment:
Haven't done a full review, but what's the new dependency for? Doesn't `redis-rs` support async out of the box? https://github.com/redis-rs/redis-rs?tab=readme-ov-file#async-support